include/onnxruntime/core/session/onnxruntime_lite_custom_op.h

```diff
@@ -18,22 +18,70 @@
 #include "onnxruntime_cxx_api.h"
 #include <optional>
 #include <numeric>
+#include <functional>
 #include <unordered_set>
 
 namespace Ort {
 namespace Custom {
 
 class TensorBase {
  public:
-  TensorBase(OrtKernelContext* ctx) : ctx_(ctx) {}
+  TensorBase(OrtKernelContext* ctx,
+             size_t indice,
```

Comment:
put "indices" as a field in tensor looks weird. is this the indices in variadic argument?

Reply:
The "indice" tells which input the current tensor is in the context.
We need it to call kernel context to fetch the corresponding input.
For output, same logic applies.
For variadic input, the "indice" will always zero because the variadic will fetch all inputs in context and keep them as one custom tensor.
For variadic output, same logic applies.

Reply:
Instand of call it "TensorBase", it is more like a "ArgumentBase". the input indices is the metadata for the argument, which can be applied to different argument type, like Tensor, or TensorList (the Varadic type)

Reply:
Done.